template/test/renderer/store.spec.js

```diff
@@ -10,6 +10,6 @@ import store from 'stores';
 
 describe('{{pascalcase name}}Store [Store]', () => {
   it('should have an initial state of {status: \'enabled\'}', () => {
-    expect(store.state.status).to.equal('enabled');
+    expect(store.getState().status).to.equal('enabled');
```

Comment:
i was testing all of our `scripts` to make sure they worked, and this didn't pass! Just need to get `state` with `getState()` it seems.